File: level1/ex11.js
function solution(array, commands) {
  const answer = [];
  
  commands.forEach(command => {
      const [i, j, k] = command;
      const number = array.slice(i - 1, j).sort((a, b) => a - b)[k - 1];
      answer.push(number);
  });

  return answer;
}

File: level1/ex11.py
def solution(array, commands):
    numbers = []
    
    for command in commands:
        [i, j, k] = command
        # ith ~ jth slice
        sliced = array[i - 1 : j]
        # sort and find kth
        number = sorted(sliced)[k - 1]
        # add to numbers
        numbers.append(number)
    
    return numbers